functions/go/set-labels/README.md

```diff
@@ -4,29 +4,27 @@
 
 <!--mdtogo:Short-->
 
-Add a list of labels to all resources.
+The `set-labels` function adds a list of labels to all resources. By default,
+the function will not only set the labels in `metadata.labels` but also a bunch
+of different places where have references to the labels. These settings are
```

Comment:
"bunch of different places" Sounds...imprecise and handwavy ;)
Please rephrase.